From 0bd8bc0873b908f4e13470933fe09f97096922da Mon Sep 17 00:00:00 2001
From: Martin Pulec <martin.pulec@cesnet.cz>
Date: Thu, 3 Feb 2022 11:01:47 +0100
Subject: [PATCH] CMakeList.txt: remove output lib name force (UNIX)

Currently the actual library name libCFHDCodec.a disagrees with the name
generated in .pc file (-lCFHDCodecStatic) so this names the library with
the default name libCFHDCodecStatic.a (similarly for shared).
---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb0ded3..f247302 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,9 +57,6 @@ if (BUILD_LIBS)
 			target_compile_options(CFHDCodecStatic PUBLIC ${COMPILER_FLAGS})
 		
 			set_target_properties(CFHDCodecStatic PROPERTIES POSITION_INDEPENDENT_CODE ON)
-			if (UNIX)
-				set_target_properties(CFHDCodecStatic PROPERTIES OUTPUT_NAME CFHDCodec)
-			endif (UNIX)
 		
 			target_link_libraries(CFHDCodecStatic)
 		endif (BUILD_SEPARATED)
-- 
2.37.3

